app/src/main/java/org/gdg/frisbee/android/api/model/ChapterDeserializer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2013-2015 The GDG Frisbee Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gdg.frisbee.android.api.model;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonDeserializer;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParseException;
+
+import java.lang.reflect.Type;
+
+public class ChapterDeserializer implements JsonDeserializer<Chapter> {
+    @Override
+    public Chapter deserialize(JsonElement jsonElement, Type type, JsonDeserializationContext jsonDeserializationContext) throws JsonParseException {
+        Chapter chapter = new Gson().fromJson(jsonElement, type);
+        chapter.name = chapter.name.trim();
```

Comment:
Do we have 'GDG'  keyword coming from the API for every chapter? In my manual tests, it seems that this PR fixes the problem described in the issue.